doc/github-org-owners.md

```diff
@@ -0,0 +1,44 @@
+## GitHub org owners
+
+The following people are owners of the GitHub Organisation:
+<!-- Keep this in sync with the code owners of github.md! -->
+- [@infinisil](https://github.com/infinisil)
+- [@lassulus](https://github.com/lassulus)
+- [@tomberek](https://github.com/tomberek)
+- [@winterqt](https://github.com/winterqt)
+- [@zimbatm](https://github.com/zimbatm)
+
+For any GitHub-related needs, you can reach out to the org owners by either
+[creating an issue](https://github.com/NixOS/org/issues/new) in this repository
+or messaging in the [Github org owners help desk Matrix room](https://matrix.to/#/#org_owners:nixos.org).
+
+### Authority and processes
+
+All org owners can individually take care of implementing:
+- Decisions by bodies that have the authority to make GitHub org changes such as:
+  - Arbitrary decisions by the Steering Committee.
+  - Moderation decisions by the moderation team.
+  - Changes from approved RFCs.
+- Low-impact changes, such as:
+  - Adding new org members to allow review requests.
+  - Creating new unprivileged Nixpkgs teams for mention.
+  - Updating repository meta information.
+  - Taking care of occasional maintenance of otherwise unmaintained repositories.
+
+With approval by at least 2 org owners, they can take care of implementing
+higher-impact changes that are _not controversial_, such as:
+- Administer unmaintained repos, such as:
+  - Giving commit access to trusted people that offer maintenance.
+  - Archiving if appropriate.
+- Create GitHub apps to unblock automation.
```

Comment:
The intent is that owners would only do maintenance with _low impact_ by themselves. It's nested under the "low-impact changes" point to imply that, but maybe that's not clear enough.

However, it's hard to estimate impact, so maybe err on the safe side and have this require 2+ approvals. Especially also because if necessary, 2+ org owners can delegate one of their own to have commit access to a specific repo and therefore empower them to maintain on their own.  

I'll move it for now, I convinced myself :)